### DISCUSSION TITLE
Where is the Windows QT version?

### DISCUSSION THREAD

**Discussion by adabo** [neovim/neovim#29260] [Q&A] [answered]
I tried looking for nvim-qt.exe from the chocolatey package manager and from the github release page but cannot find it. Has the release build stopped supporting the qt GUI?

**Comment by clason** [comment, marked as answer]
Nvim-qt is a separate project, and you can still get it from their repository (equalsraf/neovim-qt). We have stopped bundling it with our Windows releases because it's not really ours.

**Reply by adabo** [reply]
Thank you for the answer!
